sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/statefulOperators.scala

```diff
@@ -167,6 +165,18 @@ trait WatermarkSupport extends UnaryExecNode {
       }
     }
   }
+
+  protected def removeKeysOlderThanWatermark(storeManager: StreamingAggregationStateManager,
```

Comment:
incorrect indent of parameters

Reply:
Actually... where is this used? This does not seem to be used anywhere?

Reply:
Missed spot. It will be called from Update mode of StateStoreSaveExec. Will address.